src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,138 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private transient int size;
+    private transient Node<T> head;
+    private transient Node<T> tail;
```

Comment:
Why to you need the keyword `transient ` in your solution?

Reply:
Actually, I don't know. Just stupidly took this from LinkedList origin implementation without searching info. My bad)